### PR TITLE
Add Json Content based tests to `AwsSecretsLoaderTest`

### PR DESCRIPTION
**Pull request by Yogeshkumarb57**

### CODE REVIEW

File: server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java [hunk 1]
```diff
@@ -71,7 +72,7 @@ void cleanup() {
         System.clearProperty("AWS_SECRET_MANAGER_NAMES");
     }
 
-    String vaidYamlString() {
+    String validYamlString() {
```
Comment (irinaepshteyn) [inline server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java:75]: No need to have it as a method. Convert to constant

File: server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java [hunk 2]
```diff
@@ -138,7 +148,54 @@ void testCreateResourcesWithWhitespaceInSecretName() {
     }
 
     @Test
-    void testEnvironmentVariablesLoadedFromSecretName() throws JsonProcessingException {
+    void testEnvironmentVariablesLoadedFromSecretNameWithYamlContent() throws JsonProcessingException {
```
Comment (irinaepshteyn) [inline server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java:151]: testEnvironmentVariablesLoadedFromSecretNameWithYamlContent() and testEnvironmentVariablesLoadedFromSecretNameWithJsonContent() could be converted into parameterized test

```
@ParameterizedTest(name = "testEnvironmentVariablesLoadedFromSecretNameWithValidContent(): {index}")
@MethodSource("getValidContent")
void testEnvironmentVariablesLoadedFromSecretNameWithValidContent(List<Resource> resources) throws JsonProcessingException {
...
}

private static Stream<List<Resource>> getValidContent() throws JsonProcessingException {
      return Stream.of(resourcesWithValidYamlContent(), resourcesWithValidJsonContent());
}
```

File: server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java [hunk 3]
```diff
@@ -256,11 +313,47 @@ void testEnvironmentVariablesNotLoadedWithInvalidYamlContent() {
         }
     }
 
+    @Test
+    void testEnvironmentVariablesNotLoadedWithInvalidJsonContent() {
```
Comment (irinaepshteyn) [inline server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java:317]: testEnvironmentVariablesNotLoadedWithInvalidJsonContent() and testEnvironmentVariablesNotLoadedWithInvalidYamlContent() could be converted into parameterized test

```
@ParameterizedTest(name = "testEnvironmentVariablesNotLoadedWithInvalidContent(): {index}")
@MethodSource("getInvalidContent")
void testEnvironmentVariablesNotLoadedWithInvalidContent(List<Resource> resources) {
...
}

private static Stream<List<Resource>> getInvalidContent() throws JsonProcessingException {
      return Stream.of(resourcesWithInvalidYamlContent(), resourcesWithInvalidJsonContent());
}

```

**Review by irinaepshteyn — COMMENTED** [review]
@Yogeshkumarb57, please address my comments

**Comment by irinaepshteyn** [comment]
Also, consider fixing warnings like:
- Call to 'assertTrue()' from 'Assert' should be replaced with a call to a method from 'org.junit.jupiter.api.Assertions'
- 'assertTrue()' can be simplified to 'assertEquals()'

**Review by irinaepshteyn — APPROVED** [review]

File: server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java (hunk 1 above)
Comment (Yogeshkumarb57) [reply server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java]: Fixed by Irina

File: server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java (hunk 2 above)
Comment (Yogeshkumarb57) [reply server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java]: Fixed by Irina

File: server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java (hunk 3 above)
Comment (Yogeshkumarb57) [reply server/src/test/java/org/cloudfoundry/identity/uaa/impl/config/AwsSecretsLoaderTest.java]: Fixed by Irina

**Pull request merged**